lnwire/test_message.go

```diff
@@ -59,7 +59,7 @@ func (a *AcceptChannel) RandTestMessage(t *rapid.T) Message {
 	if includeLocalNonce {
 		nonce := RandMusig2Nonce(t)
 		localNonce = tlv.SomeRecordT(
-			tlv.NewRecordT[NonceRecordTypeT, Musig2Nonce](nonce),
+			tlv.NewRecordT[NonceRecordTypeT](nonce),
```

Comment:
Nice, I think the compiler advanced enough to make these no longer necessary?

Reply:
yeah and this is caught by my nvim